src/Sarif.Viewer.VisualStudio/Sarif/SarifLog.extensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.CodeAnalysis.Sarif;
+
+namespace Microsoft.Sarif.Viewer.Sarif
+{
+    /// <summary>
+    /// Extension methods for the <see cref="SarifLog"/> class used by the viewer.
+    /// </summary>
+    internal static class SarifLogExtensions
```

Comment:
>SarifLogExtensions [](start = 26, length = 18)

I know that our usual practice is to put all extension methods in a single file. At the moment there are more than a dozen per-class extension method classes, and I'm following the pattern of the existing code. It should be cleaned up all at once.